nvidia_jetson/manifest.json

```
@@ -0,0 +1,32 @@
+{
+  "display_name": "Nvidia Jetson",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "nvidia_jetson",
+  "metric_prefix": "nvidia.jetson.",
+  "metric_to_check": "",
```

include a metric to check to install tile/make dashboard available

```suggestion
  "metric_to_check": "nvidia.jetson.mem.used",
```